playground/mocks/data/idp/idx/authenticator-verification-factor-list.json

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
I think we can use https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/data/idp/idx/authenticator-verification-select-authenticator-without-signed-nonce.json instead of adding this new json file. The difference is Email instead of Password, but for the purposes of this file, what we really care about is that it has Okta Verify push